schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -258,7 +258,96 @@
+                  <auc:CoolingPlantCondition>Good</auc:CoolingPlantCondition>
+                  <auc:YearInstalled>2020</auc:YearInstalled>
+                  <auc:PrimaryFuel>Other</auc:PrimaryFuel>
+                  <auc:BuildingAutomationSystem>true</auc:BuildingAutomationSystem>
+                  <auc:ControlSystemTypes>
+                    <auc:ControlSystemType>
+                      <auc:Digital></auc:Digital>
+                    </auc:ControlSystemType>
+                  </auc:ControlSystemTypes>
+                </auc:CoolingPlant>
+              </auc:CoolingPlants>
+            </auc:Plants>
+            <auc:HeatingAndCoolingSystems>
+              <auc:ZoningSystemType>Single zone</auc:ZoningSystemType>
+              <auc:HeatingSources>
+                <auc:HeatingSource ID="HeatingSource-A">
+                  <auc:InputCapacity>123</auc:InputCapacity>
+                  <auc:OutputCapacity>123</auc:OutputCapacity>
+                  <auc:CapacityUnits>gpm</auc:CapacityUnits>
+                  <auc:HeatingSourceCondition>Good</auc:HeatingSourceCondition>
+                  <auc:Controls>
+                    <auc:Control>
+                      <auc:OtherControlTechnology>
+                        <auc:ControlSystemType>
+                          <auc:Digital></auc:Digital>
+                        </auc:ControlSystemType>
+                      </auc:OtherControlTechnology>
+                    </auc:Control>
+                  </auc:Controls>
+                  <auc:YearInstalled>2020</auc:YearInstalled>
+                </auc:HeatingSource>
+              </auc:HeatingSources>
+              <auc:CoolingSources>
+                <auc:CoolingSource ID="CoolingSource-A">
+                  <auc:CoolingSourceType>
+                    <auc:CoolingPlantID IDref="CoolingPlant-A" />
+                  </auc:CoolingSourceType>
+                </auc:CoolingSource>
+              </auc:CoolingSources>
+              <auc:Deliveries>
+                <auc:Delivery ID="Delivery-A">
+                  <auc:DeliveryType>
+                    <auc:CentralAirDistribution>
+                      <auc:AirDeliveryType>Central fan</auc:AirDeliveryType>
+                      <auc:TerminalUnit>CAV terminal box no reheat</auc:TerminalUnit>
+                      <auc:ReheatSource>None</auc:ReheatSource>
+                      <auc:FanBased>
+                        <auc:AirSideEconomizer ID="AirSideEconomizer-A">
+                          <auc:AirSideEconomizerType>None</auc:AirSideEconomizerType>
+                          <auc:EconomizerControl>Fixed</auc:EconomizerControl>
+                        </auc:AirSideEconomizer>
+                      </auc:FanBased>
+                    </auc:CentralAirDistribution>
+                  </auc:DeliveryType>
+                  <auc:HeatingSourceID IDref="HeatingSource-A" />
+                  <auc:CoolingSourceID IDref="CoolingSource-A" />
+                  <auc:Capacity>123</auc:Capacity>
```

Comment:
@nllong when we declare the capacity of an `auc:Delivery` separately from the `auc:CoolingSource` and `auc:HeatingSource`, what does that mean?  Is that non-sensical?  Should an `auc:Delivery` not declare an `auc:Capacity`?

Reply:
What would the capacity of a `auc:Delivery` be? Could that be the total design CFM?  This does seem strange.

Reply:
1. Remove requirement of `auc:Capacity` and `auc:CapacityUnits` on a Delivery system for the schematron.
2. Begin deprecation of these elements for the `auc:Delivery`.

Reply:
Addressed here ffa31a1f2145a7e7a6b57c67a03c5879c6781b46